### PR TITLE
add mist cleanup script to catalyst-api

### PR DESCRIPTION
**Pull request by emranemran**
This PR integrates the mist-cleanup script that will be called periodically to clean up /dev/shm. A separate PR will be used to actually schedule and call this script. 

Best to review on a per commit basis. @Thulinma: FYI so that you can see the diff from your original script. 

Related Catalyst PR to install the script in `build/` dir: https://github.com/livepeer/catalyst/pull/441

### CODE REVIEW

**Comment by Thulinma** [comment]
The lines originally ending in ` > /dev/null` were not meant to be logged - those would end up being _extremely_ spammy if left in. (They were used for debugging/checking the script while it was being written only.)

**Review by thomshutt — APPROVED** [review]
Structurally looks fine, will let Jaron address the content

**Review by Thulinma — DISMISSED** [review]
As long as the lines that originally went to `/dev/null` don't end up in the log, everything else looks good to me. The other changes to the script should not harm anything as far as I can see.

File: scripts/mist-cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  STRM=${P:17} #Strip the beginning
+  if containsElement "$STRM" "$ACTS" ; then
+    log "Yes: ${STRM} (${P})"
+  else
+    log "STATE: ${P}"
+    rm "${P}"
+  fi
+done
+
+log "Done. Deleted ${DATA_DEL} data pages"
```
Comment (pwilczynskiclearcode) [inline scripts/mist-cleanup.sh:82]: Is it worth calculating and displaying number of deleted semaphores and state pages?

**Comment by emranemran** [comment]
> As long as the lines that originally went to `/dev/null` don't end up in the log, everything else looks good to me. The other changes to the script should not harm anything as far as I can see.

OK, added logging levels to turn those lines off.

**Pull request merged**